### PR TITLE
OCM-9291 | test: automate id:59530 create byo oidc config cluster via rosacli

### PR DESCRIPTION
**Pull request by yuwang-RH**
[OCM-9291](https://issues.redhat.com//browse/OCM-9291) | test: automate id:59530 create byo oidc config via rosacli
log: https://privatebin.corp.redhat.com/?162137d24ab55ec4#3xh9NQkwGqhq9VEcRJRqrX54cmzgJd6UFcP3KdBrrFfo

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster.go [hunk 1]
```diff
@@ -1743,3 +1743,65 @@ var _ = Describe("create/delete operator-roles and oidc-provider to cluster",
...
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			clusterConfig, err = config.ParseClusterProfile()
+			Expect(err).ToNot(HaveOccurred())
+			awsClient, err = aws_client.CreateAWSClient("", "")
+			Expect(err).To(BeNil())
+		})
+
+		It("to check byo oidc cluster created successfully - [id:59530]",
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster.go:1770]:
```suggestion
		It("verify byo oidc cluster is created successfully - [id:59530]",
```

File: tests/e2e/test_rosacli_cluster.go [hunk 2]
```diff
@@ -1743,3 +1743,65 @@ var _ = Describe("create/delete operator-roles and oidc-provider to cluster",
...
+			labels.Critical, labels.Runtime.Day1Post,
+			func() {
+				By("Create oidc config from cluster config")
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster.go:1773]:
```suggestion
				By("Retrieve oidc config from cluster config")
```

File: tests/e2e/test_rosacli_cluster.go [hunk 3]
```diff
@@ -1743,3 +1743,65 @@ var _ = Describe("create/delete operator-roles and oidc-provider to cluster",
...
+				clusterID = config.GetClusterID()
+				oidcConfigC = clusterConfig.Aws.Sts.OidcConfigID
+
+				By("Get the operato-roles")
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster.go:1777]:
```suggestion
				By("Get the operator roles")
```

File: tests/e2e/test_rosacli_cluster.go [hunk 4]
```diff
@@ -1743,3 +1743,65 @@ var _ = Describe("create/delete operator-roles and oidc-provider to cluster",
...
+				rosaClient.Runner.JsonFormat()
+				jsonOutput, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).To(BeNil())
+				rosaClient.Runner.UnsetFormat()
+				jsonData := rosaClient.Parser.JsonData.Input(jsonOutput).Parse()
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster.go:1782]: You could use `clusterService.GetJSONClusterDescription` which does the same

**Comment by yuwang-RH** [comment]
@radtriste Could you help take a look again, all comments are updated.

**Review by radtriste — APPROVED** [review]
/lgtm

**Pull request merged**